src/library_dylink.js

```diff
@@ -299,7 +299,7 @@ var LibraryDylink = {
   _emscripten_dlopen_js: function(handle, onsuccess, onerror, user_data) {
     abort(dlopenMissingError);
   },
-  _dlsym_js: function(handle, symbol) {
+  _dlsym_js: function(handle, symbol, symbolIndex) {
```

Comment:
This is a non-functional drive-by signature fix for consistency. Looks like a new parameter was added, but was not updated here.